implementations/rust/ockam/ockam_command/src/static/preview_tooltip.txt

```diff
@@ -0,0 +1 @@
+Preview features are available for use, but are still under rapid development. There may be some minor gaps in functionality or documentation. Your feedback on the enhancements you'd like to see on this feature are especially valuable, please send them through at https://github.com/build-trust/ockam/discussions/new
```

Comment:
Love the wording here.

non-blocking change/thought; would having some newlines between important blocks of information be easier for users to digest.

e.g.

> Preview features are available for use, but are still under rapid development. 
>
> There may be some minor gaps in functionality or documentation.
>
> Your feedback on the enhancements you'd like to see on this feature are especially valuable, please send them through at https://github.com/build-trust/ockam/discussions/new

Reply:
Done! It looks like this now:

![image](https://github.com/build-trust/ockam/assets/12375782/7fb7439d-30c4-49e7-b845-e8a375f2d1ed)